packages/tailwindcss-language-server/src/project-locator.ts

```diff
@@ -500,7 +504,12 @@ async function* contentSelectorsFromCssConfig(entry: ConfigEntry): AsyncIterable
       }
     } else if (item.kind === 'auto' && !auto) {
       auto = true
-      for await (let pattern of detectContentFiles(entry.packageRoot)) {
+      let root = entry.entries[0]
```

Comment:
Took me a second to realize this was pointing to the CSS file entry (but it makes sense). I think a comment clarifying it would be useful.